libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -603,6 +603,70 @@ string YulUtilFunctions::overflowCheckedIntSubFunction(IntegerType const& _type)
 	});
 }
 
+string YulUtilFunctions::overflowCheckedIntExpFunction(
+	IntegerType const& _type,
+	IntegerType const& _exponentType
+)
+{
+	solUnimplementedAssert(!_type.isSigned(), "");
+	solAssert(!_exponentType.isSigned(), "");
+	string functionName = "checked_exp_" + _type.identifier();
```

Comment:
Does `_exponentType` need to be in the function name identifier as well?

Reply:
Good catch!